local-cluster/Cargo.toml

```diff
@@ -23,6 +23,7 @@ solana-entry = { workspace = true }
 solana-gossip = { workspace = true }
 solana-ledger = { workspace = true }
 solana-logger = { workspace = true }
+solana-net-utils = { workspace = true }
```

Comment:
Replying to this question: https://github.com/anza-xyz/agave/pull/3705#discussion_r1858914167
> side note, bind_to_localhost is used directly in LocalCluster. And while LocalCluster is technically only used for testing, it is its own standalone crate. based on my understanding, it seems like we can't make bind_to_localhost #[cfg(feature = "dev-context-only-utils")]. is that right?

I think you could if you do:
```
solana-net-utils = { workspace = true, features = ["dev-context-only-utils"] }
```
Like you said, `LocalCluster` itself is primarily testing so I think it falls in the category of core development crate
https://github.com/anza-xyz/agave/blob/81e2c0d47b010d54a215545e5ff465da5e1c8f7b/scripts/check-dev-context-only-utils.sh#L17-L20

I see some dcou in `dev-dependencies`, but given that you're modifying "source" and not tests, I don't think you could include it in dev-dep list
https://github.com/anza-xyz/agave/blob/81e2c0d47b010d54a215545e5ff465da5e1c8f7b/local-cluster/Cargo.toml#L44-L53